docs/UserGuide.md

```diff
@@ -294,22 +301,23 @@ Clears all existing data in TA-Assist.
 " %}
 
 Format: `clear`
-* Clears all existing data in TA-Assist.
-
-## Features Available in Focus Mode
+## Commands Available in Focus Mode
 
 The following commands are only available in [**focus mode**](#modes).
 
-| Command   | Format                                        |
-|-----------|-----------------------------------------------|
-| `list`    | `list`                                        |
-| `adds`    | `adds s/SESSION_NAME... [d/DATE]`             |
-| `deletes` | `deletes s/SESSION_NAME...`                   |
+| Command   | Format                                   |
+| --------- | ---------------------------------------- |
+| `list`    | `list`                                   |
+| `adds`    | `adds s/SESSION_NAME... [d/DATE]`        |
+| `deletes` | `deletes s/SESSION_NAME...`              |
 | `grade`   | `grade INDEX... s/SESSION_NAME g/GRADE_VALUE` |
-| `scores`  | `scores s/SESSION_NAME`                       |
-| `view`    | `view INDEX`                                  |
-| `export`  | `export`                                      |
-| `unfocus` | `unfocus`                                     |
+| `scores`  | `scores s/SESSION_NAME`                  |
+| `view`    | `view INDEX`                             |
+| `export`  | `export`                                 |
+| `unfocus` | `unfocus`                                |
+
+*Click [here](# Commands available in default mode) for commands available in default mode.*
```

Comment:
Maybe mention here that the commands available in default mode is also available in focus mode. See if there are exceptions; then mention them too.